github.sgnl.yaml

```diff
@@ -0,0 +1,1099 @@
+        type: String
+      - name: isBountyHunter
+        externalId: $.node.isBountyHunter
+        type: Bool
+      - name: isCampusExpert
+        externalId: $.node.isCampusExpert
+        type: Bool
+      - name: isDeveloperProgramMember
+        externalId: $.node.isDeveloperProgramMember
+        type: Bool
+      - name: isEmployee
+        externalId: $.node.isEmployee
+        type: Bool
+      - name: isFollowingViewer
+        externalId: $.node.isFollowingViewer
+        type: Bool
+      - name: isHireable
+        externalId: $.node.isHireable
+        type: Bool
+      - name: isSiteAdmin
+        externalId: $.node.isSiteAdmin
+        type: Bool
+      - name: isViewer
+        externalId: $.node.isViewer
+        type: Bool
+      - name: location
+        externalId: $.node.location
+        type: String
+      - name: name
+        externalId: $.node.name
+        type: String
+      - name: pinnedItemsRemaining
+        externalId: $.node.pinnedItemsRemaining
+        type: Int64
+      - name: projectsResourcePath
+        externalId: $.node.projectsResourcePath
+        type: String
+      - name: projectsUrl
+        externalId: $.node.projectsUrl
+        type: String
+      - name: resourcePath
+        externalId: $.node.resourcePath
+        type: String
+      - name: twitterUsername
+        externalId: $.node.twitterUsername
+        type: String
+      - name: updatedAt
+        externalId: $.node.updatedAt
+        type: DateTime
+      - name: url
+        externalId: $.node.url
+        type: String
+      - name: viewerCanChangePinnedItems
+        externalId: $.node.viewerCanChangePinnedItems
+        type: Bool
+      - name: viewerCanCreateProjects
+        externalId: $.node.viewerCanCreateProjects
+        type: Bool
+      - name: viewerCanFollow
+        externalId: $.node.viewerCanFollow
+        type: Bool
+      - name: viewerIsFollowing
+        externalId: $.node.viewerIsFollowing
+        type: Bool
+      - name: websiteUrl
+        externalId: $.node.websiteUrl
+        type: String
+  # Fetched from the 'repositories' attribute belonging to the Teams entity.
+  # TeamRepository is a Repository entity type: https://docs.github.com/en/graphql/reference/objects#repository
+  TeamRepository:
+    parent: Team
+    displayName: TeamRepository
+    externalId: $.repositories.edges
+    description: Entity representing a GitHub repository assigned to a team.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: $.node.id
+        type: String
+        indexed: true
+        uniqueId: true
+        relationshipAttributeId: TeamRepository-id
+        # Permission is the Team's permission level for this repository
+        # https://docs.github.com/en/graphql/reference/enums#repositorypermission
+      - name: permission 
```

Comment:
Same comment as above. Ingest only those attributes that `Repository` entity does not have but this one has in the context of a Team (along with unique identifier ofcourse). 

TeamRepository would be same-as Repository right?
Repository would be a superset of TeamRepository right?

Reply:
fixed